CorsixTH/Src/th_movie.cpp

```diff
@@ -307,7 +307,7 @@ void av_packet_queue::push(AVPacket* pPacket) {
     (defined(CORSIX_TH_USE_FFMPEG) &&                      \
      LIBAVCODEC_VERSION_INT < AV_VERSION_INT(57, 12, 100))
   if (av_dup_packet(pPacket) < 0) {
-    throw - 1;
+    throw -1;
```

Comment:
Different clang format versions seem to disagree on this - why am I throwing -1 anyway though? Replace with a more descriptive throw or add a no-format comment.